elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -286,11 +339,150 @@ default void updateServletContextHandler(ServletContextHandler servletContextHan
     }
 
     /**
-     * Gets the audit logger for elide
+     * Gets the audit logger for elide.
      *
      * @return Default: Slf4jLogger
      */
     default AuditLogger getAuditLogger() {
         return new Slf4jLogger();
     }
+
+    /**
+     * Gets the dynamic compiler for elide.
+     *
+     * @return Optional ElideDynamicEntityCompiler
+     */
+    default Optional<ElideDynamicEntityCompiler> getDynamicCompiler() {
+        ElideDynamicEntityCompiler dynamicEntityCompiler = null;
+
+        if (enableDynamicModelConfig()) {
+            try {
+                dynamicEntityCompiler = new ElideDynamicEntityCompiler(getDynamicConfigPath());
+            } catch (Exception e) { // thrown by in memory compiler
+                throw new IllegalStateException(e);
+            }
+        }
+
+        return Optional.ofNullable(dynamicEntityCompiler);
+    }
+
+    /**
+     * Gets the DataStore for elide.
+     * @param queryEngine query engine object.
+     * @param aggregationDataStore AggregationDataStore object.
+     * @param entityManagerFactory EntityManagerFactory object.
+     * @return EntityDictionary object initialized.
+     */
+    default DataStore getDataStore(SQLQueryEngine queryEngine, AggregationDataStore aggregationDataStore,
+            EntityManagerFactory entityManagerFactory) {
+
+        DataStore jpaDataStore = new JpaDataStore(
+                () -> { return entityManagerFactory.createEntityManager(); },
+                (em -> { return new NonJtaTransaction(em); }));
+
+        DataStore dataStore = new MultiplexManager(jpaDataStore, queryEngine.getMetaDataStore(), aggregationDataStore);
+
+        return dataStore;
+    }
+
+    /**
+     * Gets the AggregationDataStore for elide.
+     * @param queryEngine query engine object.
+     * @param optionalCompiler optional dynamic compiler object.
+     * @return AggregationDataStore object initialized.
+     */
+    default AggregationDataStore getAggregationDataStore(SQLQueryEngine queryEngine,
```

Comment:
INstead of binding this to the SQLQueryEngine - can we just bind it to QueryEngine?